sevsnpmeasure/cmd/root.go

```
@@ -19,11 +51,128 @@ func Execute() error {
 func newRootCmd() *cobra.Command {
 	rootCmd := &cobra.Command{
 		Use:   "sevsnpmeasure",
-		Short: "Calculate SNP launch measurements from OVMF binaries",
-		Long:  "Calculate SNP launch measurements from OVMF binaries.",
+		Short: "Calculate AMD SEV/SEV-ES/SEV-SNP guest launch measurement",
+		Long:  "Calculate AMD SEV/SEV-ES/SEV-SNP guest launch measurement.",
+		RunE:  launchMeasurement,
 	}
 
+	rootCmd.Flags().StringVarP(&mode, "mode", "m", "", "Guest mode, either 'snp', 'seves', 'sev', 'snp:ovmf-hash' or 'snp:svsm'.")
+	rootCmd.MarkFlagRequired("mode")
```

golangci-lint (1.62.2)

Error return value of `rootCmd.MarkFlagRequired` is not checked

(errcheck)